app/views/cases/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if current_user %>
```

Comment:
now that liang's integrated the cancan gem, you can get rid of the `current_user` check here and handle it in the controller instead